tests/SearchServiceTest.php

```diff
@@ -51,43 +49,18 @@ protected function loadIndex(int $count = 10): ServiceFake
         return $service;
     }
 
-    protected function mockClient(): Client
-    {
-        $client = $this->getMockBuilder(Client::class)
-            ->setMethods([
-                'indexDocuments',
-                'deleteDocuments',
-                'getDocuments',
-                'listDocuments',
-                'getSchema',
-                'updateSchema',
-                'listEngines',
-                'createEngine',
-            ])
-            ->disableOriginalConstructor()
-            ->getMock();
-
-        Injector::inst()->registerService($client, Client::class);
-
-        return $client;
-    }
```

Comment:
All usages of this mock should now be replaced with the more accurate API response mocking